packages/@react-aria/select/src/useSelect.ts

```diff
@@ -150,6 +150,7 @@ export function useSelect<T>(props: AriaSelectOptions<T>, state: SelectState<T>,
         }
 
         if (props.onFocus) {
+          props.onFocusChange(true);
```

Comment:
We should check `props.onFocusChange` is defined. Probably should be something like this:
```
        if (props.onFocus) {
          props.onFocus(e);
        }

        if (props.onFocusChange) {
          props.onFocusChange(true);
        }
```
Same for the ones below